releasenotes/notes/crd-webhook-v1.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+- 18771
+- 18838
+releaseNotes: |
+  *Upgraded* the CRD and Webhook versions to `v1`.
+upgradeNotes: |
+  Kubernetes 1.16+ is now required for installation.
```

generally, upgrade notes on istio.io have titles. Thoughts on this? Might not need a title at all here. 

```suggestion
  -title: Kubernetes 1.16+ required
   content: Kubernetes 1.16+ is now required for installation.
```